README.md

```diff
@@ -272,7 +273,36 @@ Incorrect:
 
     determineForeignPositionAction = 'determineForeignPosition'
 
+### Marking a list as a source only bucket
 
+Sometimes you may have a need to define a bucket of items that is "source only". This means
+that you can only grab items from it to add to other buckets. The source bucket can never be
+reordered or modified by dragging items out of it. It is only a source to drag to other lists.
+
+This list would be marked as source only:
+
+```hbs
+{{#drag-sort-list
+  items         = items1
+  dragEndAction = (action 'dragEnd')
+  sourceOnly=true
```

Comment:
Align assignment.

Reply:
Really not a fan of all these spaces, especially in hbs, but okay 👍

Reply:
Have you ever seen another Ember addon or app that does this? I have not.